arm-compute/2015-11-01-preview/swagger/containerService.json

```diff
@@ -0,0 +1,516 @@
+            "description": ""
+          },
+          "204": {
+            "description": ""
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerService/containerServices": {
+      "get": {
+        "tags": [
+          "ContainerService"
+        ],
+        "operationId": "ContainerService_List",
+        "description": "The operation to list container services.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/ContainerServiceListResult"
+            }
+          }
+        }
+      }
+    }
+    },
+    "definitions": {
+        "Resource": {
+          "properties": {
+            "id": {
+              "readOnly": true,
+              "type": "string",
+              "description": "Resource Id"
+            },
+            "name": {
+              "readOnly": true,
+              "type": "string",
+              "description": "Resource name"
+            },
+            "type": {
+              "readOnly": true,
+              "type": "string",
+              "description": "Resource type"
+            },
+            "location": {
+              "type": "string",
+              "description": "Resource location"
+            },
+            "tags": {
+              "type": "object",
+              "additionalProperties": {
+                "type": "string"
+              },
+              "description": "Resource tags"
+            }
+          },
+          "required": [
+            "location"
+          ],
+          "x-ms-azure-resource": true
+        },
+        "ContainerServiceOrchestratorProfile": {
+          "properties": {
+            "orchestratorType": {
+              "type": "string",
+              "enum": [
+                "Mesos",
+                "SwarmPreview",
+                "DCOS"
+              ],
+			  "x-ms-enum": {
+                "name": "ContainerServiceOchestratorTypes"
+              },
+              "description": "Specifies what orchestrator will be used to manage container cluster resources."
+            }
+          },
+          "description": ""
+        },
+        "ContainerServiceMasterProfile": {
+          "properties": {
+            "count": {
+              "type": "integer",
+              "format": "int32",
+              "enum": [                
+                1,
+                3,
+                5
+              ],
```

Comment:
oh yes. It is perfectly valid to leave it like this. Do file an issue in the AutoRest repo for supporting non-string enums